From d43e1333436fb61cea9d053d6999d6be74799ba3 Mon Sep 17 00:00:00 2001
From: "yang.lee" <yang.lee@starfivetech.com>
Date: Thu, 26 May 2022 11:18:32 +0800
Subject: [PATCH 2/3] clang: Use python3 in python scripts

Some scripts ask for python, but they work fine with python3
and in OE python symlink is not exposed to build, only python3 is

Rebased By Lee Yang
Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 .../find-all-symbols/tool/run-find-all-symbols.py               | 2 +-
 clang-tools-extra/clang-tidy/add_new_check.py                   | 2 +-
 clang-tools-extra/clang-tidy/rename_check.py                    | 2 +-
 clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py            | 2 +-
 clang/tools/clang-format/git-clang-format                       | 2 +-
 clang/tools/scan-view/bin/scan-view                             | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/run-find-all-symbols.py b/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/run-find-all-symbols.py
index 8655af137bb2..e95ba59a421c 100755
--- a/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/run-find-all-symbols.py
+++ b/clang-tools-extra/clang-include-fixer/find-all-symbols/tool/run-find-all-symbols.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 #=- run-find-all-symbols.py - Parallel find-all-symbols runner -*- python  -*-=#
 #
diff --git a/clang-tools-extra/clang-tidy/add_new_check.py b/clang-tools-extra/clang-tidy/add_new_check.py
index fc8397448631..0fc97862dcae 100644
--- a/clang-tools-extra/clang-tidy/add_new_check.py
+++ b/clang-tools-extra/clang-tidy/add_new_check.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 #===- add_new_check.py - clang-tidy check generator ---------*- python -*--===#
 #
diff --git a/clang-tools-extra/clang-tidy/rename_check.py b/clang-tools-extra/clang-tidy/rename_check.py
index 9c2021751e0e..4bb9af8cbe0a 100755
--- a/clang-tools-extra/clang-tidy/rename_check.py
+++ b/clang-tools-extra/clang-tidy/rename_check.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 #===- rename_check.py - clang-tidy check renamer ------------*- python -*--===#
 #
diff --git a/clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py b/clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py
index 6bd05531333b..9e134123ef44 100755
--- a/clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py
+++ b/clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 #===- clang-tidy-diff.py - ClangTidy Diff Checker -----------*- python -*--===#
 #
diff --git a/clang/tools/clang-format/git-clang-format b/clang/tools/clang-format/git-clang-format
index 1d6f216240c1..4a12070c0081 100755
--- a/clang/tools/clang-format/git-clang-format
+++ b/clang/tools/clang-format/git-clang-format
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 #===- git-clang-format - ClangFormat Git Integration ---------*- python -*--===#
 #
diff --git a/clang/tools/scan-view/bin/scan-view b/clang/tools/scan-view/bin/scan-view
index 6165432e7af8..07effbca5969 100755
--- a/clang/tools/scan-view/bin/scan-view
+++ b/clang/tools/scan-view/bin/scan-view
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 
-- 
2.25.1

